src/main/java/org/takes/http/BkBasic.java

```diff
@@ -70,17 +77,34 @@ public BkBasic(final Take tks) {
         this.take = tks;
     }
 
+//      @todo #519:30min Need to support Keep-Alive header here and take
+//       into account specified timeout and max requests parameters.
+//       See http://tools.ietf.org/id/draft-thomson-hybi-http-timeout-01.html
+//       for details. Also need to remove input.available <= 0 condition
+//       - this is done in order to prevent failure of some tests,
+//       should be replaced with proper Keep-Alive header in tests
     @Override
+    @SuppressWarnings({
```

Comment:
@exper0 annotation should be formatted (at least with current qulice) like:
```
@SuppressWarnings({ "PMD.EmptyCatchBlock", "PMD.AvoidInstantiatingObjectsInLoops" })
```
or, if the line length is to big:
```
@SuppressWarnings({ "PMD.EmptyCatchBlock",
    "PMD.AvoidInstantiatingObjectsInLoops" })
```